### ISSUE TITLE
Markdown

### ISSUE THREAD

**Issue by michaelcadilhac** [nvquanghuy/texpaste#5]
_Feature request_: Add some Markdown syntax.  I'm sure this is just a matter of including the right javascripts :-)

Cheers.

**Comment by freeboson** [comment]
Yes, specifically, support for lists/enumerations etc. would be very good.
